src/task_queue/database_tasks/media_library_cleanup.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+from django.http import HttpResponse
+from _main_.utils.emailer.send_email import send_massenergize_email_with_attachments
+from api.store.common import (
+    find_duplicate_items,
+    get_admins_of_communities,
+    get_duplicate_count,
+    remove_duplicates_and_attach_relations,
+    summarize_duplicates_into_csv,
+)
+from api.utils.constants import MEDIA_LIBRARY_CLEANUP_TEMPLATE
+from database.models import FeatureFlag
+
+
+REMOVE_DUPLICATE_IMAGE_FLAG_KEY = "remove-duplicate-images-feature-flag"
+
+
+def remove_duplicate_images():
+    """
+    This checks all media on the platform and removes all duplicates.
+    Its based on the "Remove Duplicate Images" feature flag. For communities that are subscribed
+    to the flag, duplicates will be removed from their libraries.
+    """
+    try: 
+        flag = FeatureFlag.objects.filter(key=REMOVE_DUPLICATE_IMAGE_FLAG_KEY).first()
+
+        for community in flag.communities.all():
```

Comment:
this only works if you specify communities in the flag.  See other examples:
communities = Community.objects.filter(is_deleted=False)
enabled_communities = flag.enabled_commnities(communities)

Reply:
Oooh thats true! 🤣 🤣  I will modify that! 🤝